src/release/breaking-changes/boxdecoration-shape-migration.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Migration guide for BoxDecoration.circle
+description: Removal of BoxDecoration.circle and how to migrate
+---
+
+## Summary
+
+`BoxDecoration(shape: BoxShape.circle)` was removed, and existing apps should
+use `ShapeDecoration(shape: CircleBorder())` instead.
+
+## Context
+
+According to [`Hixie comment`]):
```

Comment:
Users may not know who Hixie is. I think it is ok to include this explanation without attribution. :)